river/PointerConstraint.zig

```diff
@@ -86,11 +86,11 @@ fn constrainToRegion(self: *Self) void {
 
         if (!self.constraint.region.containsPoint(cx, cy, &box)) {
             var nRects: c_int = undefined;
-            const rects = self.constraint.region.rectangles(&nRects);
+            const rects = self.constraint.region.rectangles();
 
             if (nRects > 0) {
```

Comment:
`nRects` is now `undefined`, you need to instead check if `rects.len > 0`

Reply:
So what does nRects does now, we just need to declare it and everything happens in the background? I mean we don't need to use it, pixman handle it?

Reply:
It is no longer needed at all, if I am not mistaken. `rects` is now a slice instead of a c multi-object pointer and already contains the amount of objects it points to.

Reply:
`nRects` should be deleted, with the new `zig-pixman` api it is no longer needed. It was an out parameter to get the length o the returned data.

Reply:
Ok thanks for the explanation, done!